### PR TITLE
Check if syntax tree is present

### PR DESCRIPTION
**Pull request by jmarolf**

### CODE REVIEW

**Comment by jmarolf** [comment]
test vsi please

**Comment by jmarolf** [comment]
@dotnet/roslyn-ide

File: src/EditorFeatures/Test2/IntelliSense/CompletionServiceTests.vb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        Private Class TestCompletionProvider
+            Inherits CompletionListProvider
+            Public Sub New()
+            End Sub
+
+            Public Overrides Function IsTriggerCharacter(text As SourceText, characterPosition As Int32, options As OptionSet) As [Boolean]
+                Return True
+            End Function
+
+            Public Overrides Function ProduceCompletionListAsync(context As CompletionListContext) As Task
```
Comment (jasonmalinowski) [inline src/EditorFeatures/Test2/IntelliSense/CompletionServiceTests.vb]: Should this produce an item to verify that we do still call the producer in this case?

File: src/EditorFeatures/Test2/IntelliSense/CompletionServiceTests.vb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                    options:=Nothing,
+                    providers:={New TestCompletionProvider()},
+                    cancellationToken:=Nothing)
+                Assert.Null(list)
+            End Using
+        End Function
+
+        Private Class TestCompletionProvider
+            Inherits CompletionListProvider
+            Public Sub New()
```
Comment (jasonmalinowski) [inline src/EditorFeatures/Test2/IntelliSense/CompletionServiceTests.vb]: Delete?

File: src/EditorFeatures/Test2/IntelliSense/CompletionServiceTests.vb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            <Workspace>
+                <Project Language="NoCompilation" AssemblyName="TestAssembly" CommonReferencesPortable="true">
+                    <Document>
+                        var x = {}; // e.g., TypeScript code or anything else that doesn't support compilations
+                    </Document>
+                </Project>
+            </Workspace>
+            Using workspace = Await TestWorkspace.CreateAsync(workspaceDefinition)
+                Dim document = workspace.CurrentSolution.Projects.First.Documents.First
+                Dim completionService As AbstractCompletionService = New CSharpCompletionService()
```
Comment (jasonmalinowski) [inline src/EditorFeatures/Test2/IntelliSense/CompletionServiceTests.vb]: Seems odd this is saying CSharpCompletionService when we're not on a CSharp document.

**Pull request closed without merging**